english/madness/writhe/writhe.md

```diff
@@ -1,4 +1,12 @@
 You writhe in agony for ten thousand millenia until 
 sweet death descends upon you.
 
-WEEP MORTAL!
\ No newline at end of file
+WEEP MORTAL!
+
+But it was only a dream.  You wake soaked with sweat with a bucket of chicken next to the bed.  In the corner of the room, a camera watches silently.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.